src/DependencyInjection/HandlerMapping/TagBasedMapping.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\DependencyInjection\HandlerMapping;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+abstract class TagBasedMapping implements HandlerMapping
+{
+    public function build(ContainerBuilder $container, Routing $routing): Routing
```

Comment:
do we really build ? and what do we build ? As far I understand, we use this mapping to register or to route buses, no ? I really like the design but not the name :p

Reply:
Yeah, this was mainly generic naming while I slapped together a few different iterations of this. 

At a high level, we're building the routing, i.e. what commands go to what handlers on what buses. The actual construction and configuring of buses is all done elsewhere. I'll try to add a more complex example soon, the current one is so simple as to be confusing.